sphinx/source/docs/user_guide.rst

```diff
@@ -63,6 +63,9 @@ topic:
 :ref:`userguide_geo`
     Working with geographical data - Google Maps, GeoJSON, Tile Rendering.
 
+:ref:`userguide_graphs`
+    Create network graph visualizations with configurable node and edge interactions.
+
```

Comment:
I had to intention of asking about ordering and totally spaced it. I'll make the change now.